packages/components/message/src/Message.comp.tsx

```diff
@@ -0,0 +1,116 @@
+import { defineComponent, onBeforeUnmount, onMounted, SetupContext, isVNode, h, computed } from 'vue'
+import IxIcon from '@idux/components/icon/src/Icon.vue'
+import { PropTypes, isUndefined } from '@idux/cdk/utils'
+import { MessageItemProps, useGlobalConfig } from '@idux/components'
+import { Subject } from '@idux/cdk/subject'
+import { MessageType } from './types'
+
+export default defineComponent({
+  name: 'IxMessageItem',
+  components: {
+    IxIcon,
+  },
+  props: {
+    duration: PropTypes.number,
+    pauseOnHover: PropTypes.bool,
+    icon: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).def(''),
+    content: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).def(''),
+    type: PropTypes.oneOfType([PropTypes.oneOf(['success', 'error', 'info', 'warn'] as const), PropTypes.string]).def(
```

Comment:
```suggestion
    type: PropTypes.oneOf(['success', 'error', 'info', 'warn'] as const),
```
没必要支持 string 吧，好像没意义。